sdk/core/azure-core/azure/core/pipeline/policies/_authentication.py

```diff
@@ -121,6 +121,8 @@ def send(self, request: "PipelineRequest") -> "PipelineResponse":
                 if "WWW-Authenticate" in response.http_response.headers:
                     request_authorized = self.on_challenge(request, response)
                     if request_authorized:
+                        # if we receive a challenge response, clear the 'insecure_domain_change' tag
```

Comment:
Can we please expand this comment to explain what this tag is and what policy it's used by.